build.gradle.kts

```diff
@@ -0,0 +1,91 @@
+import com.dicedmelon.gradle.jacoco.android.JacocoAndroidUnitTestReportExtension
+import com.jfrog.bintray.gradle.BintrayExtension
+import com.novoda.gradle.release.PublishExtension
+import org.jetbrains.kotlin.gradle.dsl.Coroutines
+import org.jetbrains.kotlin.gradle.dsl.KotlinJvmProjectExtension
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+buildscript {
+    apply(from = "versions.gradle.kts")
+
+    repositories {
+        mavenCentral()
+        jcenter()
+        google()
+    }
+
+    dependencies {
+        classpath("com.android.tools.build:gradle:3.1.3")
+        classpath("com.dicedmelon.gradle:jacoco-android:0.1.3")
+        classpath("com.novoda:bintray-release:0.8.0")
+        classpath(kotlin("gradle-plugin", version = "${extra["kotlinVersion"]}"))
+    }
+}
+
+plugins { java }
+
+allprojects {
+    apply(from = rootProject.projectDir.resolve("versions.gradle.kts"))
+
+    repositories {
+        mavenCentral()
+        jcenter()
+        google()
+    }
+}
+
+subprojects {
+    val isAndroidModule = project.name == "fuel-android" || project.name == "fuel-livedata"
+    val isSample = project.name.contains("sample")
```

Comment:
Shall we add `"sample-java"` too?

Reply:
Sorry, I'm not sure what you mean. The `isSample` variable is used to not apply the Bintray release plugin in the samples modules. Also now it's specified in the change 9cb8e7a.